packages/ui/ToggleItem/ToggleItem.js

```diff
@@ -15,26 +15,69 @@ import kind from '@enact/core/kind';
 import EnactPropTypes from '@enact/core/internal/prop-types';
 import PropTypes from 'prop-types';
 import compose from 'ramda/src/compose';
-import React from 'react';
+import React, {Fragment} from 'react';
 
 import ComponentOverride from '../ComponentOverride';
 import ForwardRef from '../ForwardRef';
+import Slottable from '../Slottable';
 import Toggleable from '../Toggleable';
 import Touchable from '../Touchable';
 
 import componentCss from './ToggleItem.module.less';
 
+const ItemIcon = kind({
```

Comment:
@Djspaceg , Is is correct you expected? As you said, we don't need to handle with specific method.